modules/raft/src/main/java/org/apache/ignite/internal/raft/RaftGroupServiceImpl.java

```diff
@@ -734,14 +733,6 @@ private void handleErrorResponse(CompletableFuture<? extends NetworkMessage> fut
                 break;
             }
 
-            case EREORDER:
-                assert resp.maxObservableSafeTimeViolatedValue() != null :
-                        "Unexpected combination of EREORDER error type and null in maxObservableSafeTimeViolatedValue.";
-
-                fut.completeExceptionally(new SafeTimeReorderException(resp.maxObservableSafeTimeViolatedValue()));
```

Comment:
SafeTimeReorderException is still in use in comments
```
    /**
     * Wrapper for the update(All)Command processing result that besides result itself stores actual command that was processed. It helps to
     * manage commands substitutions on SafeTimeReorderException where cloned command with adjusted safeTime is sent.
     */
```

Reply:
🆗 
Also fixed a bug - commit ts was not propagated in case !isPrimaryInPeersAndLearners